UI/MapboxNavigationUI/TurnArrowView.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+import MapboxDirections
+import MapboxNavigation
+import SDWebImage
+
+@IBDesignable
+public class TurnArrowView: UIView {
+    var imageView: UIImageView!
+    
+    var showsShield = true
+    public var step: RouteStep? {
+        didSet {
+            imageView.isHidden = true
+            if showsShield, let components = step?.codes?.first?.components(separatedBy: " "), components.count > 1 {
+                let network = components[0]
+                let number = components[1]
+                if var imageName = ShieldImageNamesByPrefix[network] {
+                    imageName = imageName.replacingOccurrences(of: " ", with: "_").replacingOccurrences(of: "{ref}", with: number)
+                    let url = URL(string: "https://commons.wikimedia.org/w/thumb.php?f=\(imageName)&w=\(imageView.bounds.width * UIScreen.main.scale)")
```

Comment:
Frankly, our use of [thumb.php](https://www.mediawiki.org/wiki/Manual:Thumb.php) is more problematic than our use of the Wikimedia Commons API. One problem is that thumb.php has no `maxage` parameter like the API does. Does SDWebImage do a good job of avoiding duplicate requests for the same thumbnail image as the user scrolls a particular row in and out?